inst/AmyloGraph/www/amylograph.css

```diff
@@ -145,22 +133,16 @@ body {
   font-weight: bold;
 }
 
-.ag_protein_details {
-  height: calc(100% - 90px);
-}
-
-.ag_protein_details > .tabbable {
-  max-height: 70%;
-  width: 100%;
-  overflow: scroll;
+#protein_details > * {
+  overflow: auto;
 }
 
-.ag_close_button {
+#btn_close_tab {
```

Comment:
I checked the ID and it was `"close_interaction btn_close_tab"`, but I suppose there should be no space, rather a "-".